platform/linux/README.md

```diff
@@ -66,10 +66,7 @@ Set an access token as described below, and then run:
 
 ### Test
 
-- `make test-*` Builds and runs all tests. You can specify individual tests by replacing * with their name (e.g. `make test-Sprite.CustomSpriteImages`).
```

Comment:
Wouldn't it be better to update this example rather than removing it?
`make run-test-*` can be used to run test individually.

Reply:
Thanks, done, PR updated.